.github/workflows/test-s390x.yml

```diff
@@ -0,0 +1,92 @@
+name: test-s390x
+on:
+  push:
+    branches:
+      - main
+      - 'release/**'
+  pull_request:
+    branches:
+      - main
+      - 'release/**'
```

Comment:
If I understand correctly, secrets won't be exposed on pull requests from forks, so will this workflow always fail in those cases?

Reply:
https://github.com/buildpacks/lifecycle/pull/1142#discussion_r1345275208
You can check the condtion on the comment in the above link. That skips the execution when running from fork.